pcb/rules/klc_constants.py

```diff
@@ -0,0 +1,23 @@
+
+# silkscreen layer
+KLC_SILK_WIDTH = 0.12
+KLC_SILK_WIDTH_ALLOWED = [0.10, 0.12, 0.15]
```

Comment:
whould we really "allow" these? Why not just check for 0.12mm ... otherwise we will never get a consistent lib (I think)